subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/generator/TestProjectGenerator.groovy

```diff
@@ -76,6 +76,10 @@ class TestProjectGenerator {
                 file projectDir, "src/test/java/${packageName}/Test${it}.java", fileContentGenerator.generateTestClassFile(subProjectNumber, it, dependencyTree)
             }
         }
+
+        if (isRoot) {
+            file projectDir, "buildSrc/src/main/java/Thing.java", "public class Thing {}"
```

Comment:
This is just to force snapshotting of the `gradleApi`. A lot of users use buildSrc, so I think it's appropriate that we test for it in more scenarios.

Reply:
Could you add a corresponding comment above this line in the source code so this knowledge is not forgotten?

Reply:
Done